source/features/deep-reblame.css

```
@@ -2,6 +2,6 @@ button.rgh-deep-reblame .octicon-versions {
 	color: #e40de48c;
 }
 
-.rgh-deep-reblame:hover {
-	color: #8000808c;
+[aria-label*='PR first'].rgh-deep-reblame:hover {
+	color: #8000808c !important;
```

The rule above applies it to the icon itself. A parent can never override the children.

```suggestion
.rgh-deep-reblame:hover .octicon-versions {
	color: #8000808c;
```

Now we can also drop `!important` since the icon's color doesn't have have `!important`

---

Sorry I'm testing something. Duplicate review follows:

The rule above applies it to the icon itself. A parent can never override the children.

```suggestion
.rgh-deep-reblame:hover .octicon-versions {
	color: #8000808c;
```

Now we can also drop `!important` since the icon's color doesn't have have `!important`

---

>The rule above applies it to the icon itself. A parent can never override the children.
👍